deployments/keypairs.nix

```diff
@@ -1,7 +1,39 @@
 with (import ./../lib.nix);
 
-{
+let
+  regions = [
```

Comment:
Yes, but it requires some refactoring. The good news is, if regions mismatch you'll get an error :)